spec/plugins/cors_spec.lua

```diff
@@ -0,0 +1,220 @@
+local spec_helper = require "spec.spec_helpers"
+local http_client = require "kong.tools.http_client"
+local cjson = require "cjson"
+
+local APIS = {}
+local PLUGINS = {}
+local API_URL = spec_helper.API_URL
+local PROXY_URL = spec_helper.PROXY_URL
+
+function request (name, method, qs, headers)
```

Comment:
All those functions should be declared `local`. Also the `(` has no space before in all our code.